server/TypeDBService.java

```diff
@@ -59,17 +64,29 @@ public class TypeDBService extends TypeDBGrpc.TypeDBImplBase {
 
     private static final Logger LOG = LoggerFactory.getLogger(TypeDBService.class);
 
+    private final String address;
     protected final TypeDB.DatabaseManager databaseMgr;
     private final ConcurrentMap<UUID, SessionService> sessionServices;
 
-    public TypeDBService(TypeDB.DatabaseManager databaseMgr) {
+    public TypeDBService(InetSocketAddress address, TypeDB.DatabaseManager databaseMgr) {
+        this.address = address.getHostString() + ":" + address.getPort();
         this.databaseMgr = databaseMgr;
         sessionServices = new ConcurrentHashMap<>();
     }
 
     @Override
-    public void databasesContains(CoreDatabaseManager.Contains.Req request,
-                                  StreamObserver<CoreDatabaseManager.Contains.Res> responder) {
+    public void serversAll(ServerManager.All.Req request, StreamObserver<ServerManager.All.Res> responder) {
+        try {
+            responder.onNext(ResponseBuilder.ServerManager.allRes(address));
+            responder.onCompleted();
+        } catch (RuntimeException e) {
+            LOG.error(e.getMessage(), e);
+            responder.onError(exception(e));
+        }
+    }
```

Comment:
A new request that the core TypeDB now supports. The `user` requests will respond with the standard gRPC "not implemented" exception.

Reply:
We should throw our own for the ones that are not supported by this TypeDB:
```
User management is only available in TypeDB Enterprise.
```

Reply:
Ie. throw the most specific exception possible to help the user's intention